src/core_plugins/kibana/server/routes/api/remote_info/index.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { callWithRequestFactory } from './call_with_request_factory';
+
+async function fetchRemoteClusters(callWithRequest) {
+  const options = {
+    method: 'GET',
+    path: '_remote/info'
+  };
+  const remoteInfo = await callWithRequest('transport.request', options);
+  return Object.keys(remoteInfo);
+}
+
+export function registerClustersRoute(server) {
+  server.route({
+    path: '/api/kibana/clusters',
+    method: 'GET',
+    handler: async request => {
+      const callWithRequest = callWithRequestFactory(server, request);
+      try {
+        return await fetchRemoteClusters(callWithRequest);
+      } catch (error) {
+        return error.body;
```

Comment:
could we re-use [handleEsError](https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/server/lib/handle_es_error.js)?